sdk/appconfiguration/perf-tests/app-configuration/README.md

```diff
@@ -0,0 +1,9 @@
+### Guide
+
+1. Build the app-config perf tests package `rush build -t perf-app-configuration`.
+2. Copy the `sample.env` file and name it as `.env`.
+3. Create a storage account and populate the `.env` file with `APPCONFIG_CONNECTION_STRING` variable.
+4. Run the tests as follows
+
+   - list settings
+     - `npm run perf-test:node -- ListSettingsTest --warmup 2 --duration 7 --iterations 2 --parallel 2`
```

Comment:
is this service rate-limited?

Reply:
There is throttling similar to Form recognizer or metrics advisor.

Reply:
if the operation is not a LRO, can you experiment with the number for the parallel param if the limit is not too bad?

Reply:
The limit is 20,000 requests per hour, it is quite simple to hit the limit.
We'll soon get the proxy-tool so that we don't have to hit the live service all the time.